tests/kernel/irq_lock/src/main.c

```diff
@@ -0,0 +1,13 @@
+#include <ztest.h>
+
+void test_prevent_interruption(void);
+void test_prevent_interruption1(void);
```

Comment:
dont use numbers, call the test with what it actually tests.

Also, move everything to one single file, no need to have main.c and another file with the test.